src/odemis/driver/orsay.py

```diff
@@ -1389,3 +1440,1097 @@ def terminate(self):
             self._gis.PrecursorType.Unsubscribe(self._updatePrecursorType)
             self._temperaturePar = None
             self._gis = None
+
+
+class OrsayParameterConnector:
+    """
+    Attribute that is connected to a VA and a parameter on the Orsay server.
+    If VA is readonly, the VA will be kept up to date of the changes of the Orsay parameter, but force writing to the VA
+    will not update the Orsay parameter.
+    If VA is not readonly, writing to the VA will write this value to the Orsay parameter's Target attribute.
+    """
+
+    def __init__(self, va, parameter, attributeName="Actual", conversion=None, factor=None, minpar=None, maxpar=None):
+        """
+        va is the vigilant attribute this Orsay parameter connector should be connected to. This VA should not have a
```

Comment:
This docstring does not follow the default convention of docstings. In odemis we use two styles I would suggest to choose one of those.

https://www.python.org/dev/peps/pep-0257/